src/utils.js

```diff
@@ -208,6 +197,31 @@ export function getDefaultFormState(_schema, formData, definitions={}) {
   return formData || defaults;
 }
 
+export function getUiOptions(uiSchema) {
+  // get all passed options from ui:widget, ui:options, and ui:<optionName>
+  return Object.keys(uiSchema).filter(key => /^ui:/.test(key)).reduce((options, key) => {
```

Comment:
We don't necessarily care about ie but I can understand that could become a concern. `key.indexOf("ui:") === 0` should work exactly the same and be more efficient than using a regex.